### ISSUE TITLE
great work!

### ISSUE THREAD

**Issue by martindb** [nm156/CNCInfusion#2]
Hi, I´m planning to make a litle cnc laser engraver (Like thisone http://www.instructables.com/id/Pocket-laser-engraver/) to enter this wonderfull world of CNC machines. (Currently waiting arduino/drivers)

Your GRBL GUI looks realy great!! I would like to use it. There is any compiled version that I can download?

Thanks in advance!

**Comment by mike-holmes** [comment]
Yep - go to the repository home page and click the "download zip" button towards the top left. Extract everything to a folder on your computer and go to src->CNCInfusion->bin->Debug->CNCInfusion.exe.

nm156 - I echo martindb's sentiment; I look forward to testing this out on my machine on Monday.

**Comment by nm156** [comment]
Thanks for the comments.  This is a work in progress, not yet a fully ready GUI.  I just wanted to get some concept code out there to test.  Your feedback is appreciated, I should have some time this weekend to work on it some more.

**Comment by martindb** [comment]
Thank you Mike! I'll try it this weekend in the breadboard with leds... I've my arduino right now.... still waiting controllers and looking for the parts for my cnc laser engraver.
